### DISCUSSION TITLE
How to open the .pkl file in pycharm?

### DISCUSSION THREAD

**Discussion by 1400107712** [community/community#119724] [New to GitHub]
![屏幕截图 2024-04-16 054321](https://github.com/community/community/assets/167079881/0a95dfa1-c21a-4ee3-91b2-3b0ef436a1ca)
Like this picture,I don't know what should I do

**Comment by gurjeetsinghvirdee** [comment]
Hey @1400107712 

You can go through this: **https://www.folkstalk.com/2022/10/open-pkl-file-python-with-code-examples.html**
It will solve your issue
